### DISCUSSION TITLE
Welcome to ngraph-gtk Discussions!

### DISCUSSION THREAD

**Discussion by htrb** [htrb/ngraph-gtk#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.
  * You can use English or Japanese. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kidssa** [comment]
コメント失礼します．ngraphGTKについての質問です．
NgraphGTKで作成したngpファイルを前のNgraph(Ngraph for Windows:6.03.54)で開こうとすると「axis:field not found A 」と出てきてしまいます．開く方法はありますでしょうか．

**Reply by htrb** [reply]
申し開けありませんが、色々と機能追加している関係で簡単に開く方法はありません。

ngp ファイルをエディタで編集して古いバージョンで使われていない設定を削除するなどの修正により開くことができるようになる可能性がありますが、フォントの指定や数式変換など古いバージョンで対応していない機能を使用している場合はそのような部分も修正する必要があります。
